lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -1061,4 +1061,27 @@ public static function networkDeviceTypesFunction(array $params, $template)
             . $options
             . '</select>';
     }
+
+    public static function identitytypesFunction(array $params, $template)
```

Comment:
Jeśli camelcase to nie powinno być `identityTypesFunction()`?

Reply:
@chilek też o tym pomyślałem i chciałem zrobić camelcase, ale zauważyłem, że paytypesFunction jest bez więc zastosowałem się do już istniejącej 'normy' :) zaraz poprawię.

Reply:
Dzięki za pochwałkę :)

Reply:
Na początku miesiąca trzeba chwalić, a na koniec ganić :D